applications/SwimmingDEMApplication/custom_elements/calculate_laplacian_simplex_element.cpp

```diff
@@ -55,18 +55,12 @@ int ComputeLaplacianSimplex<TDim, TNumNodes>::Check(const ProcessInfo& rCurrentP
     int ErrorCode = Kratos::Element::Check(rCurrentProcessInfo);
     if(ErrorCode != 0) return ErrorCode;
 
-    if(this->GetGeometry().size() != TDim+1)
-        KRATOS_THROW_ERROR(std::invalid_argument,"wrong number of nodes for element", this->Id());
-
-    if(VELOCITY_LAPLACIAN.Key() == 0)
-
-        KRATOS_THROW_ERROR(std::invalid_argument,"VELOCITY_LAPLACIAN Key is 0. Check if the application was correctly registered.","");
+    KRATOS_ERROR_IF(this->GetGeometry().size() != TDim+1)<< "Wrong number of nodes for element" << this->Id() << std::endl;
+    KRATOS_ERROR_IF(VELOCITY_LAPLACIAN.Key() == 0)<< "VELOCITY_LAPLACIAN Key is 0. Check if the application was correctly registered." << std::endl;
 
     // Check that the element's nodes contain all required SolutionStepData and Degrees of freedom
-    for(unsigned int i=0; i<this->GetGeometry().size(); ++i)
-    {
-        if(this->GetGeometry()[i].SolutionStepsDataHas(VELOCITY_LAPLACIAN) == false)
-            KRATOS_THROW_ERROR(std::invalid_argument,"missing VELOCITY_LAPLACIAN variable on solution step data for node ",this->GetGeometry()[i].Id());
+    for(unsigned int i=0; i<this->GetGeometry().size(); ++i) {
+        KRATOS_ERROR_IF(this->GetGeometry()[i].SolutionStepsDataHas(VELOCITY_LAPLACIAN) == false)<< "Missing VELOCITY_LAPLACIAN variable on solution step data for node " << this->GetGeometry()[i].Id() << std::endl;
```

Comment:
`kratos/includes/checks.h` has macros for this

Reply:
Thanks! Done for variables and dofs.